mqtt/mqtt-bridge/src/persist/waking_state/waking_store.rs

```diff
@@ -0,0 +1,109 @@
+#![allow(dead_code)] // TODO remove when ready
+
+use std::{collections::hash_map::Entry, collections::HashMap, task::Waker};
+
+use bincode::{self};
+use mqtt3::proto::Publication;
+use rocksdb::{IteratorMode, Options, DB};
+use uuid::Uuid;
+
+use crate::persist::{waking_state::StreamWakeableState, Key, PersistError};
+
+/// When elements are retrieved they are added to the in flight collection, but kept in the original db store.
+/// Only when elements are removed from the in-flight collection they will be removed from the store.
+pub struct WakingStore {
+    db: DB,
+    in_flight: HashMap<Key, Publication>,
+    waker: Option<Waker>,
+    column_family: String,
+}
+
+impl WakingStore {
+    pub fn new(mut db: DB) -> Result<Self, PersistError> {
+        let column_family = Uuid::new_v4().to_string();
```

Comment:
I think `new` should take the column family name as an argument

Reply:
What is the problem with generating a new uuid inside new?